lib/engine/game/base.rb

```diff
@@ -993,17 +1005,18 @@ def init_hexes(companies, corporations)
 
       def init_tiles
         self.class::TILES.flat_map do |name, val|
-          if val.is_a?(Integer)
-            count = val
+          if val.is_a?(Integer) || val == 'unlimited'
+            count = val == 'unlimited' ? 1 : val
```

Comment:
Thinking on how readable this construct is, and on line 1019. Would 
 count = (val == 'unlimited' ? 1 : val) 
improve things?

Reply:
Another possibility would be to add a private
  def unlimited?(val)
     val == 'unlimited'
  end
and then use this on line 1009, 1019 and 1029.